tests/dummy/app/templates/demos/sl-calendar.hbs

```diff
@@ -30,35 +30,35 @@
 <h3>Properties</h3>
 
 <div class="list-group">
-    {{#property-text name="action" type="function"}}
+    {{#property-text name="action" type="Function"}}
         Action to call when a day is clicked that contains one of the content dates. An array of content objects with dates occurring on the clicked date will be returned to this action.
     {{/property-text}}
 
-    {{#property-text name="content" type="mixed"}}
+    {{#property-text name="content" type="Mixed"}}
```

Comment:
I do not see what you are referring to when I view https://github.com/softlayer/sl-ember-components/blob/develop/addon/components/sl-calendar.js#L180-L185